pkg/cmd/roachprod/main.go

```diff
@@ -1053,16 +1060,17 @@ default cluster settings. It's intended to be used in conjunction with
 `,
 	Args: cobra.ExactArgs(1),
 	Run: wrap(func(cmd *cobra.Command, args []string) error {
-		clusterName, err := verifyClusterName(args[0])
```

Comment:
Could we keep our use of `verifyClusterName`, which gives us nice errors+suggestions if improper? Something like splitting args[0] on ":" should suffice, and passing the first argument to verify.